lib/stdlib/src/uri_string.erl

```diff
@@ -297,16 +297,14 @@
       NormalizedURI :: uri_string()
                      | error().
 normalize(URIMap) ->
-    try normalize(URIMap, [])
-    catch
-        throw:{error, Atom, RestData} -> {error, Atom, RestData}
-    end.
+    normalize(URIMap, []).
```

Comment:
uri_string_SUITE:normalize_negative was failing in our CI due to this change.
```
{error,invalid_utf8,<<0,0,0,246>>} = uri_string:normalize("//%00%00%00%F6")
```
To make it more complicated I found problems even with the original code. The following examples trigger uncaught exceptions:
```
uri_string:normalize("//%00%00%00%F6", [return_map])
uri_string:normalize(#{host => "%00%00%00%F6",path => []}, [return_map]).
```
So if we keep the scope of this PR, I would propose to keep the try catch.

Reply:
Seems that `uri_string:normalize("/%XX")` would also trigger a throw.

I'll put the try/catch back.